packages/bitcore-wallet-service/src/lib/model/log-devide.ts

```diff
@@ -0,0 +1,93 @@
+import _ from 'lodash';
+
+const Uuid = require('uuid');
+const Defaults = require('../common/defaults');
+
+const MAX_ATTENDANCE_WEEK = 7;
+
+export interface ILogDevice {
+  id: number;
+  version: number;
+  createdOn: number;
+  updatedOn: number;
+  platform: string;
+  packageName: string;
+  token: string;
+  deviceId: string;
+  location: string;
+  isFirstInstall: boolean;
+  countNumber: number;
+}
+export class LogDevice {
+  id: number;
+  version: number;
+  createdOn: number;
+  updatedOn: number;
+  platform: string;
+  packageName: string;
+  token: string;
+  deviceId: string;
+  location: string;
+  isFirstInstall: boolean;
+  countNumber: number;
+
+  static create(opts) {
+    opts = opts || {};
+
+    const now = Math.floor(Date.now() / 1000);
+
+    const x = new LogDevice();
+
+    x.id = Uuid.v4();
```

Comment:
It same works well base on code implement before
*Example:
<img width="982" alt="Screenshot 2023-05-05 at 17 44 18" src="https://user-images.githubusercontent.com/48612433/236438141-c5baf6b3-01d7-4e63-9465-a57f68025731.png">
<img width="873" alt="Screenshot 2023-05-05 at 17 43 54" src="https://user-images.githubusercontent.com/48612433/236438170-d6cd0d7e-4ea7-4e56-a854-c8700d1172fb.png">

Reply:
I think we need to debug and check the result of Uuid.v4(). It may generate a string then be casted to number, which the data is lost in the process (128 bits -> 57 bits)

Reply:
resolve